commands/notify.js

```diff
@@ -137,10 +138,159 @@ async function DisplayListSelector (interaction, guild, question) {
+
+  // check if list exists
+  if (!Object.keys(guild.notifyLists).includes(listName)) {
+    await reply(interaction, `De lijst ${listName} bestaat niet, ${foemp()}!`)
+    return
+  }
+
+  // check if user is actually subscribed to list
+  if (!guild.notifyLists[listName].includes(userId)) {
+    await reply(interaction, `Je ontvangt geen notificaties voor ${listName}, ${foemp()}!`)
+    return
+  }
+
+  // subscribe list
+  const index = guild.notifyLists[listName].indexOf(userId)
+  guild.notifyLists[listName].splice(index, 1)
+  await saveGuild(guild)
+
+  // reply to user
+  await reply(interaction, `Je ontvangt niet langer notificaties voor de lijst ${listName}`)
+}
+
+async function notifyList (interaction) {
+  const guild = await getGuild(interaction.guildId)
+  const userId = interaction.member.user.id
+  const listName = interaction.options.getString('name')
+  const message = interaction.options.getString('message')
+
+  // check if list exists
+  if (!Object.keys(guild.notifyLists).includes(listName)) {
+    await reply(interaction, `De lijst ${listName} bestaat niet, ${foemp()}!`)
+    return
+  }
+
+  // check wheter or not message is too long
+  if (message && message.length > EMBED_MAX_FIELD_LENGTH) {
+    await reply(interaction, `Je bericht is ${message.length - EMBED_MAX_FIELD_LENGTH} karakters te lang, ${foemp()}`)
+    return
+  }
+
+  const subscribers = guild.notifyLists[listName]
+
+  // check if user is subscribed to the list
+  if (!subscribers.includes(userId)) {
+    const actionRow = new MessageActionRow()
+      .addComponents(
+        new MessageButton()
+          .setCustomId('notify')
+          .setLabel('Notify!')
+          .setStyle(BUTTON_STYLES.PRIMARY),
+        new MessageButton()
+          .setCustomId('sub')
+          .setLabel('Schrijf me in!')
+          .setStyle(BUTTON_STYLES.SUCCESS),
+        new MessageButton()
+          .setCustomId('cancel')
+          .setLabel('Annuleren')
+          .setStyle(BUTTON_STYLES.DANGER)
+      )
+
+    const initialInteraction = interaction
+    const confirmationMessage = await reply(interaction, { content: `Je bent zelf niet ingeschreven op de lijst '${listName}'. Wens je toch een notify te sturen? Of wil je enkel jezelf toevoegen?`, components: [actionRow] })
+    interaction = await confirmationMessage.awaitMessageComponent({ filter: i => i.user.id === interaction.user.id, componentType: 'BUTTON', time: DEFAULT_TIMEOUT, ephemeral: true })
+
+    // handle interaction
+    await defer(interaction)
+    initialInteraction.deleteReply()
+    if (interaction.customId === 'cancel') {
+      await reply(interaction, 'Ale, dag en bedankt dan eh!')
+      return
+    } else if (interaction.customId === 'sub') {
+      guild.notifyLists[listName].push(userId)
+      await reply(interaction, `Je ontvangt nu notificaties voor de lijst ${listName}`)
+      return
+    } else {
+      // continue and notify for list
+    }
```

Comment:
is something missing here?  if no, then :
```suggestion
    } 
```